src/contrib/cluster/Akka.Cluster.Sharding/Akka.Cluster.Sharding.csproj

```diff
@@ -5,9 +5,9 @@
     <AssemblyTitle>Akka.Cluster.Sharding</AssemblyTitle>
     <Description>Sharded actors with managed lifecycle for Akka.NET cluster</Description>
     <VersionSuffix>beta</VersionSuffix>
-    <TargetFrameworks>$(NetFrameworkLibVersion);$(NetStandardLibVersion)</TargetFrameworks>
```

Comment:
Need to resolve https://github.com/akkadotnet/akka.net/issues/3851 before doing this